sdk/batch/microsoft-azure-batch/CHANGELOG.md

```diff
@@ -1,15 +1,28 @@
 # Release History
 
-## 10.2.0-beta.1 (Unreleased)
+## 11.0.0 (2023-05-23)
 
-### Features Added
+### Features
+
+- Added a new enum `CriCompatible` to type `ContainerType`.
```

Comment:
I would place the `ContainerType` change under Breaking changes because modeling the enum as a string now changes the ContainerConfiguration.withType signature to accept a parameter of type `ContainerType`